### PR TITLE
Fixes the Invite friends button during onboarding

### PR DESCRIPTION
**Pull request by colinta**
Also fixes the "Scroll to last comment" bug.

### CODE REVIEW

File: Ello.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -4405,7 +4405,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "diff \"${PODS_ROOT}/../Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [ $? != 0 ] ; then\n    # print error to STDERR\n    echo \"error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\" >&2\n    exit 1\nfi\n";
+			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-Specs/Pods-Specs-resources.sh\"\n";
```
Comment (steam) [inline Ello.xcodeproj/project.pbxproj]: Did coapods change this?

Comment (colinta) [inline Ello.xcodeproj/project.pbxproj]: It did, and it seems to flip back and forth on 1.20 and swift3.

**Comment by colinta** [comment]
Merged latest 1.20, gonna run specs now.

**Comment by colinta** [comment]
```
Test Suite 'All tests' passed at 2017-01-17 10:21:39.861.
	 Executed 2594 tests, with 0 failures (0 unexpected) in 305.503 (306.818) seconds
```

**Pull request merged**